tests/test_widgets.py

```diff
@@ -185,6 +185,34 @@ class W(SuffixedMultiWidget):
         }, {}, 'price')
         self.assertEqual(result, ['1', 'lt'])
 
+    def test_value_omitted_from_data(self):
+        class A(SuffixedMultiWidget):
+            suffixes = ['_b']
+
+        a = A(widgets=[BooleanWidget])
+
+        result = a.value_omitted_from_data(['a', 'b', 'c'], './.', 'test')
+
+        self.assertIsNotNone(result)
+
+    def test_replace_name(self):
+        class A(SuffixedMultiWidget):
+            suffixes = ['test']
+
+        a = A(widgets=[BooleanWidget])
```

Comment:
The widget class is not used below and an instance seems needed if it was used. How about `None` or `mock.Mock()` for simplicity?